tests/internal/cluster/cluster.go

```diff
@@ -193,13 +193,14 @@ func ExecCmdWithRetries(client *clients.Settings, retries uint,
 	return wait.PollUntilContextTimeout(
 		context.TODO(), interval, time.Duration(retries-1)*interval, true, func(ctx context.Context) (bool, error) {
 			err := ExecCmd(client, nodeSelector, command)
-
 			if isErrorExecuting(err) {
-				glog.V(90).Infof("Error during command execution, retry %d (%d max): %w", retry, retries, err)
+				glog.V(90).Infof("Error during command execution, retry %d (%d max): %v", retry, retries, err)
 
 				retry++
 
 				return false, nil
+			} else if err != nil {
```

Comment:
Why else if ?

Reply:
Oh this can just be a separate if since the first block always returns, I'll update it